kiwi/builder/disk.py

```diff
@@ -184,35 +163,17 @@ def create(self) -> Result:
         Image types which triggers this builder are:
 
         * image="oem"
-        * image="vmx"
 
         :return: result
 
         :rtype: instance of :class:`Result`
         """
-        disk = DiskBuilder(
-            self.xml_state, self.target_dir, self.root_dir, self.custom_args
-        )
-        result = disk.create_disk()
+        result = self.create_disk()
+        result = self.create_install_media(result)
+        self.append_unpartitioned_space()
+        return self.create_disk_format(result)
 
-        # cleanup disk resources taken prior to next steps
-        del disk
-
-        disk_installer = DiskBuilder(
-            self.xml_state, self.target_dir, self.root_dir
-        )
-        result = disk_installer.create_install_media(result)
-
-        disk_format = DiskBuilder(
-            self.xml_state, self.target_dir, self.root_dir
-        )
-
-        disk_format.append_unpartitioned_space()
-        result = disk_format.create_disk_format(result)
-
-        return result
-
-    def create_disk(self) -> Result:
+    def create_disk(self):
```

Comment:
It still returns Result, doesn't it ?

Reply:
By bad, I took the changes from my branch on this method while solving the conflicts that raised after adding types master and I did not see that I should have picked the method signature. Thanks for catching it